content/en/docs/tutorials/clusters/seccomp.md

```diff
@@ -0,0 +1,346 @@
+* Learn how to create fine-grained seccomp profiles
+* Learn how to apply a container runtime default seccomp profile
+
+## {{% heading "prerequisites" %}}
+
+In order to complete all steps in this tutorial, you must install
+[kind](https://kind.sigs.k8s.io/docs/user/quick-start/) and
+[kubectl](../../tasks/tools/install-kubectl). We will show examples with both
+`alpha` (pre-v1.19) and GA seccomp functionality, so make sure that your cluster
+is [configured
+correctly](https://kind.sigs.k8s.io/docs/user/quick-start/#setting-kubernetes-version)
+for the version you are using.
+
+<!-- steps -->
+
+## Create Seccomp Profiles
+
+We will discuss the contents of these profiles later on, but for now go ahead
+and download them into a directory named `profiles/` so that we can load them
+into our cluster.
+
+{{< codenew file="pods/security/seccomp/profiles/audit.json" >}}
+
+{{< codenew file="pods/security/seccomp/profiles/violation.json" >}}
+
+{{< codenew file="pods/security/seccomp/profiles/fine-grained.json" >}}
+
+## Create a Local Kubernetes Cluster with Kind
+
+For simplicity, we will use [kind](https://kind.sigs.k8s.io/) to create a single
+node cluster with our seccomp profiles loaded. Kind runs Kubernetes in Docker,
+so each node of the cluster is actually just a container. This allows for files
+to be mounted in the filesystem of each container just as one might load files
+onto a node.
+
+{{< codenew file="pods/security/seccomp/kind.yaml" >}}
+
+Download the example above, and save it to a file named `kind.yaml`. Then create
+the cluster with the configuration.
+
+```
+kind create cluster --config=kind.yaml
+```
+
+Once the cluster is ready, identify the container running as the single node
+cluster.
+
+```
+docker ps
+```
+
+You should see output indicating that a container is running with name
+`kind-control-plane`.
+
+```
+CONTAINER ID        IMAGE                  COMMAND                  CREATED             STATUS              PORTS                       NAMES
+6a96207fed4b        kindest/node:v1.18.2   "/usr/local/bin/entr…"   27 seconds ago      Up 24 seconds       127.0.0.1:42223->6443/tcp   kind-control-plane
+```
+
+If we take a look at the filesystem of that container, we should see that our
+`profiles/` directory has been successfully loaded into the default seccomp path
+of the kubelet.
+
+```
+docker exec -it 6a96207fed4b ls /var/lib/kubelet/seccomp/profiles
+```
+
+```
+audit.json  fine-grained.json  violation.json
+```
+
+## Create Pod with Seccomp Profile for Syscall Auditing
+
+Next we want to apply our `audit.json` profile, which will log all syscalls of
+the process, to a new Pod. 
+
+If you are using a pre-v1.19 Kubernetes version, download this manifest as
+`audit-pod.yaml`. 
+
+{{< codenew file="pods/security/seccomp/alpha/audit-pod.yaml" >}}
+
+If using Kubernetes v1.19 or later, download this manifest as `audit-pod.yaml`.
+
+{{< codenew file="pods/security/seccomp/ga/audit-pod.yaml" >}}
```

Comment:
Consider using [tabs](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#tabs) here.

Reply:
A viable alternative: get a version of this tutorial for v1.18 merged into _master_, and then revise it for v1.19

In the v1.19 and later versions, tell the reader to look at the v1.18 version if they're using an older cluster.